client/src/components/IndexPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const IndexPage = props => {
+    const [trail, setTrail] = useState([])
+
+    const getTrails = async () => {
+        try {
+            const response = await fetch("/api/v1/index")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            setTrail(body.trail)
+        } catch (error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
```

Comment:
Looks like there's different tab indentation settings - you'll want to sync with your team and make sure each member has their `tab size` set to `2`